### DISCUSSION TITLE
RESUME_ABDURAHIM_ESHIMBEKOV

### DISCUSSION THREAD

**Discussion by lxrneo7** [lxrneo7/motion-wordle#1] [Q&A]
Give me a feedback
![Снимок экрана 2024-05-02 171644](https://github.com/darknes7sjdhfouef/motion-wordle/assets/145838102/da8eb304-a580-4ebe-ae62-070f34ad9f04)

**Comment by aziyatali** [comment]
@darknes7sjdhfouef  Overall it looks good, but there are some points we can improve.

1. Use the Template I sent you where company names are bold.
2. Add link to Github for your projects
3. For your projects, add what you did with 2-3 bullet points.
     - **designed** and **implemented** the **UI** for Bookshop website
     - **created** book loaning request, approval and returning with **Material UI**
     - **integrated** book management catalog efficiently using **React-query** 
     - Etc
     
     * Use verbs in the beginning, clearly write down what exactly you did and how you contributed using certain technologies/frameworks/languages/leadership
  4. Do **NOT** write what IT academy does but write what did you do in the IT academy. What projects you did? Follow the #3 format
  5. For education, add what relevant IT courses you took if any(such as Algorithms, Linear Algebra, OOP)


Do not forget to add links to your projects!
